frontend/src/components/ThemedButtons/ChatButton.css

```diff
@@ -9,8 +9,13 @@
 	color: var(--chat-button-text-colour);
 	font-weight: 700;
 	font-size: 1rem;
+	width: 100%;
 }
 
 .chat-button:hover {
 	background-color: var(--chat-button-background-colour-hover);
 }
+
+.chat-button-disabled:hover {
```

Comment:
You don't need an extra class for this, can use an attribute selector:

`.chat-button[aria-disabled='true']:hover {`

Reply:
ah, I knew I was missing some sort of trick!